README.md

```diff
@@ -3,7 +3,7 @@
   <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
   <br/>
```

Comment:
- Please keep in mind that the logo provided in your README file is broken. Kindly fix it 👍🏽